src/main/java/org/jitsi/stats/media/StatsServiceFactory.java

```diff
@@ -213,12 +213,6 @@ private ServerInfo createServerInfo(
 
         // the default endpoint type is server (middlebox)
         String endpointType = CallStatsConst.END_POINT_TYPE;
-
-        if (isClient)
-        {
-            endpointType = "browser";
```

Comment:
This is not needed anymore?

Reply:
Ya, Since it is server so reporting it browser is not correct.